gui/GiveAchievement.java

```diff
@@ -80,7 +94,7 @@ public void setup() {
 
 		registerComponent(new PictureButton((int) (width * .9), (int) (height * .5), 30, 30,
 				new ResourceLocation("minecraft", "textures/items/sugar.png")).setIsEnabled(true)
-						.addHoverText("Give Items").doesDrawHoverText(true)
+						.addHoverText("Remove Items").doesDrawHoverText(true)
```

Comment:
this is weird being in the give achievement gui, copy pasta on my part but should be give achievement?